src/govuk/components/error-summary/error-summary.mjs

```diff
@@ -2,8 +2,18 @@ import '../../vendor/polyfills/Function/prototype/bind.mjs'
 import '../../vendor/polyfills/Event.mjs' // addEventListener
 import '../../vendor/polyfills/Element/prototype/closest.mjs'
 
-function ErrorSummary ($module) {
+import { mergeConfigs, normaliseDataset } from '../../common.mjs'
+function ErrorSummary ($module, config) {
   this.$module = $module
+
+  var defaultConfig = {
+    disableAutoFocus: false
+  }
+  this.config = mergeConfigs(
+    defaultConfig,
+    config || {},
+    normaliseDataset($module.dataset)
```

Comment:
I think it's probably fine, but technically if someone was previously instantiating an ErrorSummary class with an `undefined` `$module` (not using `initAll`) it'd have failed silently, whereas now it throws an error.

We could add another guard inside the constructor if we wanted to play it extra safe?